src/org/labkey/test/tests/core/security/TroubleshooterRoleTest.java

```diff
@@ -0,0 +1,104 @@
+package org.labkey.test.tests.core.security;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.DailyC;
+import org.labkey.test.pages.core.admin.ShowAuditLogPage;
+import org.labkey.test.util.ApiPermissionsHelper;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({DailyC.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 4)
+public class TroubleshooterRoleTest extends BaseWebDriverTest
+{
+    private static final String TROUBLESHOOTER = "user@troubleshooter.test";
```

Comment:
Avoid display name collision
```suggestion
    private static final String TROUBLESHOOTER = "troubleshooter@troubleshooter.test";
```